### PR TITLE
Docker definitions for containers

### PR DESCRIPTION
**Pull request by akscram**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  python get-pip.py && \
+  rm -f get-pip.py
+
+COPY . /app
+WORKDIR /app
+
+RUN python setup.py install
+RUN pip install -r requirements.txt
+
+EXPOSE 5010
```
Comment (boris-42) [inline Dockerfile]: Let's use always 5000 port for all services

**Review by boris-42 — CHANGES_REQUESTED** [review]
one nit

File: Dockerfile (hunk 1 above)
Comment (akscram) [reply Dockerfile]: Done.

**Review by akscram — COMMENTED** [review]
All comments were fixed.

**Review by boris-42 — APPROVED** [review]

**Pull request merged**